pandas/_libs/tslibs/parsing.pyx

```diff
@@ -0,0 +1,688 @@
+#!/usr/bin/env python
```

Comment:
I don't think we add executable pre-amble to any other .pyx files

Reply:
Will remove.